src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
-// write code here
+const list = document.body.querySelector('ul');
+
```

Comment:
const eployees = [...list.children]
```suggestion
const eployees = [...list.children]
```
To avoid double work on 15 and 41 lines